storage/store.go

```diff
@@ -2159,11 +2185,20 @@ func (s *Store) processRaft() {
 			}
 			s.pendingRaftGroups.Unlock()
 			s.mu.Unlock()
+
+			var wg sync.WaitGroup
+			wg.Add(len(replicas))
 			for _, r := range replicas {
-				if err := r.handleRaftReady(); err != nil {
-					panic(err) // TODO(bdarnell)
-				}
+				sem.acquire()
+				go func(r *Replica) {
```

Comment:
If you're avoiding the stopper on purpose, a comment wouldn't hurt as to not make this look like an omission.